### PR TITLE
Added a Youtube Playlist to the Community Page

### LINKED ISSUE #568: Add Layer5 youtube playlists to the community page
The youtube playlists for [community meetings](https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0) and [development meetings](https://www.youtube.com/playlist?list=PL3A-A6hPO2INr4oEIorJqC8N0FtxnYj0a) should be embedded in the https://layer5.io/community page, preferable above the `engage` section.

### PR DESCRIPTION
**Pull request by Fahdmoh01**
Signed-off-by: Fahdmoh01 <fahdmoh.1@gmail.com>

**Description**

This PR fixes #568 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [*] Yes, I signed my commits.

### CODE REVIEW

**Comment by Fahdmoh01** [comment]
@Nikhil-Ladha kindly review this

**Comment by Nikhil-Ladha** [comment]
I think I have already given my suggestions in the slack chat.

**Review by Anita-ihuman — COMMENTED** [review]
👌 looking good here.

File: _includes/community/community-meetings-table.html [hunk 1]
```diff
@@ -63,3 +63,10 @@
 </table>
 <br />
 </div>
+
+
+  <div align ="center">
+    <div align="center"><h3>Checkout Our Youtube Playlist</h3></div>  
+     <iframe width=65% height="315" src="https://www.youtube.com/embed/videoseries?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (tripathyprateek) [inline _includes/community/community-meetings-table.html:70]: Increase the height a bit. The viewing ratio doesn't look good right now. Other than that great job!

**Comment by Nikhil-Ladha** [comment]
Hey @Fahdmoh01 , sorry for trouble but I think we might need to drop this idea of adding a playlist. It just doesn't feels good and goes with the site.
If in future we have any plans for this we can think about it.
Closing this PR, hope you don't mind.

**Pull request closed without merging**